net/speedtest/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+version             1.1.1.84
+revision            0
+
+name                speedtest
+categories          net
+platforms           darwin
+supported_archs     noarch
+maintainers         {@linuxgemini linuxgemini.space:macportspackaging} openmaintainer
+
+homepage            https://speedtest.net/apps/cli
+license             restrictive nomirror noncommercial
+
+description         official cli for speedtest.net
+long_description    command line interface for testing internet bandwidth using speedtest.net
+
+master_sites        https://install.speedtest.net/app/cli/
+distname            ookla-speedtest-${version}-macosx-x86_64
+extract.suffix      .tgz
+
+# ookla already ships this as an universal binary
+universal_variant   no
+supported_archs     x86_64 arm64
+
+checksums           rmd160  c16334261d6858b21ef08ebb4886ced3c303420e \
+                    sha256  153f76e4e502a2bb470c7f7c3b4b8afaa50eb98bb11e86216fcb1a8b92676e94 \
+                    size    1019149
+
+use_configure       no
+build {}
+
+destroot {
+    set installdir ${prefix}/bin
+
+    xinstall -d ${destroot}${installdir}
+    xinstall -m 755 -W ${workpath} \
+        speedtest \
+        ${destroot}${installdir}
+}
+
+post-destroot {
+    set docdir ${prefix}/share/man/man5
+
+    xinstall -d ${destroot}${docdir}
+    xinstall -m 644 -W ${workpath} \
+        speedtest.5 \
+        ${destroot}${docdir}
+}
+
+# ookla does not provide a public filetree
+livecheck.type      none
```

Comment:
How did you discover that the version number was 1.1.1.84? Is there no web page where that number is listed that could be used for livecheck?

Reply:
Hi Ryan, Ookla themselves have a tap for homebrew at https://github.com/teamookla/homebrew-speedtest/blob/master/speedtest.rb

Reply:
Ok, then how did they discover that this was the latest version?